### ISSUE TITLE
[Suggestion] Close app shortcut

### ISSUE THREAD

**Issue by Aerinx** [PoE-TradeMacro/POE-TradeMacro#142]
Could you please add a shortcut to close the app? I can do it quick and fast with default AutoHotkey reassignments, but it gets deleted every update and I'm sure this would be really fast to add proper when you know the code already (and github). I use eventghost to launch and close all  my autohotkey macros in games and I need that to close it.

Thank you for your time and this wonderful app.

**Comment by Eruyome** [comment]
Yeah I suppose I could do something like that.

Reminder for myself:
- Add a global array of scripts (path/name) to ItemInfo with a function to close all these scripts.
- Add trademacro scritp (main.ahk) to this array from trademacro.ahk
- Add an optional hotkey for this to AdditionalMacros.txt

Explanation: This way it can be used and by PoE-ItemInfo while still working when using TradeMacro where ItemInfo is included to close all related scripts.

**Comment by Eruyome** [comment]
added on dev branch

**Comment by Aerinx** [comment]
Many releases have gone by since and the shortcut is not yet on the macro, what happened with this? Was it removed by some reason?

**Comment by Eruyome** [comment]
It's there since I said it's there, just in AdditionalMacros.txt.

**Comment by Aerinx** [comment]
Oh, I did not even think about looking there even when I was using it, I can see it now, thanks. Even so, I already tried to use a shortcut there for that (^!+End::ExitApp) and it is not useful.

It defeats the purpose since those shortcuts only work while the game is focused and the objective is to close the script after the game has been closed. I used to hack the shortcut into the old main.ahk if I recall correct, wich was a pita having to do it after every update. I have not done it in quite some time and the last time I tried I could not even find where I have to put it for this to work after the refactor that removed or renamed that file.
